server/ai_live_video.go

```diff
@@ -152,6 +152,9 @@ func startControlPublish(control *url.URL, params aiRequestParams) {
 		ControlPub:  controlPub,
 		StopControl: stop,
 	}
+	if monitor.Enabled {
```

Comment:
I think we should probably move this initialisation of the LivePipeline somewhere more central now that we're using it to check stream existence as well as just storing these control related fields. Also do we need a mutex around it like we do in `cleanupLive()` @j0sh ?